src/main/java/com/ernie/TicketApp/ApplicationContext.java

```diff
@@ -25,6 +32,7 @@ private static void loadProperties() {
         Properties properties = new Properties();
         try (FileInputStream input = new FileInputStream("src/main/resources/application.properties")) {
             properties.load(input);
+            dbClassName = properties.getProperty("db.className");
             dbUrl = properties.getProperty("db.url");
             dbUsername = properties.getProperty("db.username");
             dbPassword = properties.getProperty("db.password");
```

Comment:
do not load properties with the help of `FileInputStream` and ` Properties`. Use 1 of `Spring` approaches:
1. add `@PropertySource("classpath:application.properties")`
   and read values with `@Value`:
```java
 @Value("${db.url}")
 private String dbUrl;
   ... 
    dataSource.setUrl(dbUrl);
   ...
 ```

2. add `@PropertySource("classpath:application.properties")`
   and read values with `Spring Environment`:
```java
private final Environment env;
public ApplicationContext(Environment env) {this.env = env;}
   ... 
    dataSource.setUrl("db.url"));
   ...
 ```